packages/block-library/src/form-submit-button/block.json

```diff
@@ -0,0 +1,17 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "core/form-submit-button",
+	"title": "Form submit button",
+	"category": "common",
+	"icon": "button",
+	"parent": [ "core/form" ],
+	"description": "A submission button for forms.",
+	"keywords": [ "submit", "button", "form" ],
+	"textdomain": "default",
+	"supports": {
+		"lock": false,
+		"multiple": false
```

Comment:
It would be nice if the `supports` flag worked like this where `multiple` for a block that has a `parent` means that only once per parent element. 

But unless that changed decently and I missed it that is not the case. So this here means that there can only ever be one form on a page.

Reply:
This is a bit of a weird mix of behaviors:
The user can add multiple forms in a page, and they all work fine (perhaps because they get added from the form's template). Each form has its own submit button as expected, so all good here. If they try to add a 2nd submit button in a form, they can't - so far so good.
The bug in the behavior is when they delete a submit-button from a form, and try to add it again. In that case, they cannot.

I'll go ahead and remove the restriction from block.json to keep things simple 👍

Reply:
Just for reference I think this here is a perfect example why #38958 would be great to have :)